### PR TITLE
docs(assets): Add sample code for org policy analyzer

### PR DESCRIPTION
**Pull request by Christopher312**
## Description

Add sample code for org policy analyzer

## Checklist
- [x] I have followed guidelines from [CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md) and [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [x] **Tests** pass:   `npm test` (see [Testing](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#run-the-tests-for-a-single-sample))
- [x] **Lint** pass:   `npm run lint` (see [Style](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#style))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new sample directory, and I created [GitHub Actions workflow](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#adding-new-samples) for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Comment by Christopher312** [comment]
I granted permission to ` jenkins@nodejs-docs-samples-tests.iam.gserviceaccount.com`, is that the right account?

File: asset/snippets/analyzeOrgPolicies.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+'use strict';
+
+// sample-metadata:
+//   title: Analyze Org Policies
+//   description: Analyzes accessible Org policies that match a request.
+//   usage: node analyzeOrgPolicies
+
+async function main(scope, constraint) {
+  // [START asset_quickstart_analyze_org_policies]
+
```
Comment (kweinmeister) [inline asset/snippets/analyzeOrgPolicies.js:24]: Nit: start region tag one line lower to avoid top line being blank in docs

**Review by kweinmeister — APPROVED** [review]

File: asset/snippets/analyzeOrgPolicies.js (hunk 1 above)
Comment (Christopher312) [reply asset/snippets/analyzeOrgPolicies.js]: Thanks, I removed the extra line

File: asset/snippets/test/orgPolicyAnalyzer.test.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+const {Storage} = require('@google-cloud/storage');
+
+const {BigQuery} = require('@google-cloud/bigquery');
+const bigquery = new BigQuery();
+const options = {
+  location: 'US',
+};
+const datasetId = `asset_nodejs_${uuid.v4()}`.replace(/-/gi, '_');
+
+const orgId = 'organizations/474566717491'; // This is the id of ipa1.joonix.net, a test organization
```
Comment (rsamborski) [inline asset/snippets/test/orgPolicyAnalyzer.test.js:34]: Please add PoC for the org in the comment.

**Review by rsamborski — CHANGES_REQUESTED** [review]
Minor nit asking for an org PoC. Additionally please make sure `kokoro-system-test@long-door-651.iam.gserviceaccount.com` has proper permissions for CI tests to pass.

**Review by rsamborski — APPROVED** [review]

**Comment by rsamborski** [comment]
@Christopher312 please confirm that `kokoro-system-test@long-door-651.iam.gserviceaccount.com` has proper permissions for CI tests to pass. If you did grant these permissions and tests keep failing let me know.

**Comment by Christopher312** [comment]
> @Christopher312 please confirm that `kokoro-system-test@long-door-651.iam.gserviceaccount.com` has proper permissions for CI tests to pass. If you did grant these permissions and tests keep failing let me know.

Yes it does. The tests are passing, the one check in the PR that failed is because this is from a forked repository, not because of the tests themselves.

**Pull request merged**